Packages/com.unity.inputsystem/Documentation~/KnownLimitations.md

```diff
@@ -0,0 +1,20 @@
+# Known Limitations
+
+The following is a list of known limitations that the Input System currently has.
+
+## Compatibility with other Unity features
+
+* `PlayerInput` split-screen support does not work with Cinemachine virtual cameras.
+* The Input System cannot generate input for IMGUI or UIElements.
+* The Input System does not yet support the new 2019.3 mode where domain reloads are disabled when entering play mode.
+
+## Device support
+
+* (Windows) Mouse input from remote desktop connections does not come through properly.
+* (Windows) Pen input will not work with Wacom devices if "Windows Ink" support is turned off.
+* Joy-Cons are only supported on Switch.
```

Comment:
Should the comments about Switch and PS4 clarify that support for this is in separate packages?

Reply:
Done